client/components/UserDecalPanel.js

```diff
@@ -87,6 +90,37 @@ class UserDetailsPanel extends Component {
     return null
   }
 
+  _renderDecalCode = (decal, index) => (
+    <>
```

Comment:
The problem with that is it messes up the formatting, the grid requires all direct child elements to function correctly (sub-grids, when?), otherwise the date column would either have variable widths for each row, or a fixed-width, not based on the content.